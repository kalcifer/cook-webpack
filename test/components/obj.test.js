import React from 'react';
import renderer from 'react-test-renderer';

import Obj from '../../components/obj';

test('Should be object component', () => {
    const children = 'Strings';
    const component = renderer.create(<Obj description="This is test component" children={children}/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})