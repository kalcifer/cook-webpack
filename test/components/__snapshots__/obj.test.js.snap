exports[`test Should be object component 1`] = `
<div
  data-type="obj">
  This is test component
  Strings
</div>
`;
